app/components/item/Item.js

```diff
@@ -919,14 +912,7 @@ export default class Item extends Component<Props, State> {
                   {item.genres && <h6>{item.genres.join(', ')}</h6>}
                 </span>
               </Row>
-              {/* HACK: Prefer a CSS solution to this, using text-overflow: ellipse */}
-              <h6 className="row-margin" id="summary">
```

Comment:
The e2e tests rely on the `#summary` element to exist. If you want to fix the tests you'll need to fix the query selector in the tests

Reply:
Completely missed that 👍